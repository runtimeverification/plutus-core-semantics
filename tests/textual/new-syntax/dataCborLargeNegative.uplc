(program 1.0.0 (con data #c349010000000000000000))
