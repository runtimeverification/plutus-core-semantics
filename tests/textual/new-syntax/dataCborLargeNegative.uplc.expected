      ( con data { Integer -18446744073709551617 } ) 